newsroom/companies/__init__.py

```diff
@@ -1,39 +1,32 @@
+import superdesk
+
 from flask_babel import lazy_gettext
 
-import superdesk
+from newsroom.companies.companies_async.types import CompanyResource
+from newsroom.users.model import UserResourceModel
+
 from superdesk.core import json, get_current_app
 from apps.prepopulate.app_initialize import get_filepath
 
-from newsroom.auth import get_company
 from newsroom.user_roles import UserRole
 
 from .companies import CompaniesResource, CompaniesService
 from .module import module  # noqa
 
 
-def get_company_sections_monitoring_data(company_id, user):
+async def get_company_sections_monitoring_data(company: CompanyResource, user: UserResourceModel):
     """get the section configured for the company"""
     app = get_current_app().as_any()
-    if not company_id or user["user_type"] == UserRole.ADMINISTRATOR.value:
-        return {"userSections": app.sections}
-
-    company = superdesk.get_resource_service("companies").find_one(req=None, _id=company_id)
 
-    rv = {
-        "monitoring_administrator": (company or {}).get("monitoring_administrator"),
-        "userSections": app.sections,
-    }
-    if company and company.get("sections"):
-        rv["userSections"] = [s for s in app.sections if company.get("sections").get(s["_id"])]
+    if not company or user.user_type == UserRole.ADMINISTRATOR.value:
```

Comment:
When checking against an enum, there is no longer the need to use `.value` from the enum attribute.

i.e.
```console
>>> from newsroom.user_roles import UserRole
>>> from pydantic import BaseModel
>>> 
>>> class User(BaseModel):
...     name: str
...     role: UserRole
... 
>>> user = User(name="Mark", role=UserRole.PUBLIC)
>>> user.role
<UserRole.PUBLIC: 'public'>
>>> user.role == UserRole.PUBLIC
True
>>> user.role == UserRole.INTERNAL
False
```